presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoColumnHandle.java

```diff
@@ -31,18 +33,23 @@
     public static final String SAMPLE_WEIGHT_COLUMN_NAME = "presto_sample_weight";
 
     private final String name;
+    private final String alias;
```

Comment:
do we need a new field?

can we just pass original (un-lower-cased) `name` into `ColumnMetadata#properties`, as it's done in https://github.com/prestosql/presto/pull/1914 ?

Reply:
After reviewing the code yes it is redundant, have removed it.